tests/hazmat/primitives/test_rsa.py

```diff
@@ -963,8 +965,9 @@ def test_signature_not_bytes(self, backend):
         signature = 1234
 
         with pytest.raises(TypeError):
-            public_key.verifier(
+            public_key.verify(
```

Comment:
Looks like I went overboard here and lost some coverage: https://codecov.io/gh/pyca/cryptography/compare/e722e557e08171e265008711b824df44d4e9ec7d...9238a6369b657305706426ee24a8bb64d4024065/src/src/cryptography/hazmat/backends/openssl/rsa.py